Sources/FluentUI_iOS/Core/Theme/Tokens/ControlTokenSet.swift

```diff
@@ -204,9 +206,11 @@ public class ControlTokenSet<T: TokenSetKey>: ObservableObject {
 /// Union-type enumeration of all possible token values to be stored by a `ControlTokenSet`.
 public enum ControlTokenValue {
     case float(() -> CGFloat)
-    case uiColor(() -> UIColor)
     case color(() -> Color)
+#if os(iOS) || os(visionOS)
+    case uiColor(() -> UIColor)
```

Comment:
I'm assuming macOS will exclusively use swift color then? (no plans to add a nsColor control token). if so any reason to keep uiColor around (besides the fact that deprecating it will be a pain)

Reply:
> I'm assuming macOS will exclusively use swift color then? (no plans to add a nsColor control token). 
For the time being, that's the goal. We'll see if there's customer feedback asking for NSColor, in which case we can consider it.

> I'm assuming macOS will exclusively use swift color then? (no plans to add a nsColor control token). if so any reason to keep uiColor around (besides the fact that deprecating it will be a pain)
100%, it's just a deprecation thing we'd have to stage and plan and roll out. Maybe someday!